pkgs/top-level/aliases.nix

```diff
@@ -166,9 +166,7 @@ mapAliases ({
   emacsPackagesNgGen = emacsPackagesFor; # added 2018-08-18
   emacsPackagesNgFor = emacsPackagesFor; # added 2019-08-07
   emacsPackagesNg = emacs.pkgs; # added 2019-08-07
-  emacs26Packages = emacs26.pkgs; # added 2020-12-18
   emacs27Packages = emacs27.pkgs; # added 2020-12-18
-  emacs26WithPackages = emacs26.pkgs.withPackages; # added 2020-12-18
```

Comment:
Shouldn't these be converted into throws?

Reply:
isn't attribute doesn't exist error enough? I would say in the end the result won't be that much better.

Reply:
What we've typically done in the past is to remove old dropped version specific attributes.